src/functions-and-arrays.js

```diff
@@ -1,19 +1,67 @@
 // Iteration #1: Find the maximum
-
+let maxOfTwoNumbers = (num1, num2) => {
+  if (num1 > num2) {
+    return num1
+  }
+  if (num1 < num2) {
+    return num2
+  }
+  return num1
+}
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+let findLongestWord = (arrayOfWords) => {
+  if (arrayOfWords === [] || arrayOfWords === ['']){
+    return null
+  } else {
+    let biggest = 0;
+    let longestWord = '';
+
+    for (i = 0; i < arrayOfWords.length; i++) {
+      word = arrayOfWords[i]
+      switch (maxOfTwoNumbers(biggest, word.length)) {
+        case biggest: 
+          break;
+        case word.length:
+          biggest = word.length;
+          longestWord = word;
+      }
+    }
+    return longestWord
+  }
+}
 // Iteration #3: Calculate the sum
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+sumNumbers = (numberArray) => {
+  sum = 0;
+  for (i = 0; i < numberArray.length; i++) {
+    sum += numberArray[i];
+  }
+  return sum
+}
 // Iteration #4: Calculate the average
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+averageNumbers = (arrayNumbers) => {
+  average = sumNumbers(arrayNumbers)/arrayNumbers.length
+ return average
```

Comment:
you forgot to do a if statement when an array is empty and return null
example:
```
if(arr.length==0){
return null
}
```

Reply:
Thanks a lot!  Yeah, I noticed that this morning during the review. 

Thanks for the feedback.  It gives closure.  :D